src/core_modules/capture-ui/FlatList/FlatList.component.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import React from 'react';
+import cx from 'classnames';
+import uuid from 'uuid/v4';
+import classes from './FlatList.module.css';
```

Comment:
I am wondering if we could also use the `withStyles` which come from the `import { withStyles } from '@material-ui/core';`. 

Or better you might wanna help me understand why you went with this alternative in this case? :)

Reply:
I agree, this is removed on another branch 10948